src/icons.rs

```diff
@@ -37,12 +37,29 @@ fn no_icons() -> Value {
         "volume_empty": " VOL ",
         // This icon has no spaces around it because it is manually set as text. (sound.rs)
         "volume_muted": "MUTED",
-        "thermometer": " TEMP "
-    })
+        "thermometer": " TEMP ",
+    });
+
+    if cfg!(feature = "weather") {
+        let mut icons = icons.as_object_mut().unwrap();
+        let weather = [
+            ("weather_sun", " OUT "),
+            ("weather_snow", " OUT "),
+            ("weather_thunder", " OUT "),
+            ("weather_clouds", " OUT "),
+            ("weather_rain", " OUT "),
+            ("weather_default", " OUT ")
```

Comment:
Are you sure about the OUT description? Could just output Sun, Snow, Thunder, etc in no_icons mode.